contributing/code_of_conduct/enforcement_team.rst

```diff
@@ -0,0 +1,66 @@
+Enforcement team
+================
+
+The role of the enforcement team is to make sure the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
+is respected. If you think there is or has been a violation to the code of conduct
+please read the :doc:`reporting guidelines </contributing/code_of_conduct/reporting_guidelines>`
+and contact any person in the enforcement team.
+
+Members
+-------
+
+Lukas Kahwe Smith
+`````````````````
+.. image:: https://avatars3.githubusercontent.com/u/300279?s=400&v=4
+   :height: 150px
+   :width: 150px
+   :alt: Likas Kahwe Smith
+   :class: pull-right
+
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, susciit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+Lukas Kahwe Smith
+`````````````````
+.. image:: https://avatars3.githubusercontent.com/u/300279?s=400&v=4
+   :height: 150px
+   :width: 150px
+   :alt: Likas Kahwe Smith
+   :class: pull-right
+
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, suscipit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+Lukas Kahwe Smith
+`````````````````
+.. image:: https://avatars3.githubusercontent.com/u/300279?s=400&v=4
+   :height: 150px
+   :width: 150px
+   :alt: Likas Kahwe Smith
+   :class: pull-right
+
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, suscipit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+About the Enforcement Team
+--------------------------
+
+The :doc:`Symfony core team </contributing/code/core_team>` appoints enforcement
```

Comment:
Can we get some more details about the enforcement team members? For how long are they elected? Can their appointments be renewed? For how long or for how many times? etc.

Reply:
Great questions. I do not have any answers to these questions. I *felt* that we *assumed* that members are elected until they resign.

Reply:
We kind of left that process to the core team. I also think that generally I would expect this to not to be a time limited position (similar to the core team itself). And yes the core team could also remove someone. Not sure if we really want to include that process in here, but if someone feels strongly it should I am open to adding it.

I guess the question is then, does it require a majority, 2/3 majority, unanimous? is there a quorum? what is the vote duration? I do think that the vote itself should be secret from the outside, ie. the core team just presents the results but not who voted what.

Reply:
If we read what happens after, it seems tricky that's the core team's duty to select the enforcement team since alot of the core team is part / working with sensiolabs.

Reply:
I think the percentage of core team people working for sensiolabs is actually not quite that high and slowly decreasing.